go-magic-link/README.md

```diff
@@ -0,0 +1,59 @@
+# Go-admin-portal-example
+An example Golang application demonstrating how to use the [WorkOS Golang SDK](https://github.com/workos-inc/workos-go) so your customers can access the WorkOS Admin Portal from your application.
+
+
+## Go Project Setup
+
+1. In your CLI, navigate to the directory into which you want to clone this git repo.
+   ```bash
+   $ cd ~/Desktop/
+   ```
+
+2. Clone the example repo using your preferred secure method (HTTPS or SSH).
+   ```bash
+   # HTTPS
+   $ git clone https://github.com/workos-inc/Go-Example-Applications.git
+   ```
+
+   or
+
+   ```bash
+   # SSH
+   $ git clone git@github.com:workos-inc/Go-Example-Applications.git
+   ```
+
+3. Navigate to the cloned repo and to the admin portal:
+   ```bash
+   $ cd Go-Example-Applications/go-magic-link
+   ```
+
+4. Obtain and make note of the following values. In the next step, these will be set as environment variables.
+   - Your [WorkOS API key](https://dashboard.workos.com/api-keys)
+
+5. Add this to your APIKey variable.
```

Comment:
Is it difficult with Go to set env variables? If it is, then no worries, but might be a little more slick with a .env file.